### PR TITLE
ISPN-11339 adding details about remote clients

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-11339

Needs backport to 1.1.0

### CODE REVIEW

**Comment by rigazilla** [comment]
[exposing_infinispan.txt](https://github.com/infinispan/infinispan-operator/files/4218225/exposing_infinispan.txt)
@oraNod I've wrote my notes on clients in the attachment.

File: documentation/asciidoc/stories/assembly_connecting_remotely.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+The examples in this section use ${SERVICE_HOSTNAME} to denote the cluster IP
+address or route hostname that provides access to your {brandname} cluster.
+
+|===
+|${SERVICE_HOSTNAME} |Client location
+
+|`.spec.clusterIP`
+|Inside {k8s}.
+
+|`.spec.clusterIP`
```
Comment (galderz) [inline documentation/asciidoc/stories/assembly_connecting_remotely.adoc]: Where did you get this from? From the simple tutorial?

For the external service it'd depend on the type of external service and even the provider. E.g. for a load balancer on AWS, that's `.status.loadBalancer.ingress\[0\].hostname`, whereas in Google/Azure I believe that'd be  `.status.loadBalancer.ingress\[0\].ip`. In NodePort it'd be actual nodes' host/port (in minikube, `minikube ip`...etc, not tried in other envs). `.spec.clusterIP` is really only relevant if you access it from inside Kubernetes, but in that case you can just use the service name itself, along with the port.

Comment (oraNod) [reply documentation/asciidoc/stories/assembly_connecting_remotely.adoc]: thanks @galderz this is a nice catch. yeah I think I was getting a bit mixed up and probably started working on this section, didn't quite finish, but didn't catch it before I sent the PR. updated now.

Comment (rigazilla) [reply documentation/asciidoc/stories/assembly_connecting_remotely.adoc]: @galderz I would like to publish docs for 1.1.1 (already on hub). Can you approve if all is ok?

**Comment by rigazilla** [comment]
@oraNod, please resolve conflicts. I would like to merge if ok

File: documentation/asciidoc/stories/assembly_connecting_remotely.adoc (hunk 1 above)
Comment (oraNod) [reply documentation/asciidoc/stories/assembly_connecting_remotely.adoc]: @rigazilla I asked @galderz for review on these changes and he confirmed they look good. I'll resolve this conversation now. should be good to merge.

**Comment by rigazilla** [comment]
merged, thanks @oraNod

**Pull request merged**